apps/docs/src/app/core/component-docs/select/select-docs.component.html

```diff
@@ -1,107 +1,131 @@
-<fd-docs-section-title [id]="'standard'" [componentName]="'select'">
-    Standard Select
+<fd-docs-section-title [id]="'mode'" [componentName]="'select'">
+    Select Modes
 </fd-docs-section-title>
-<description>A basic example of the select component with no advanced features.</description>
+<description>
+    Following examples presents different modes of the Select control.
+</description>
 <component-example>
-    <fd-select-basic-example></fd-select-basic-example>
+    <fd-select-mode-example></fd-select-mode-example>
 </component-example>
-<code-example [exampleFiles]="selectBasic"></code-example>
+<code-example [exampleFiles]="selectMode"></code-example>
 
 <separator></separator>
 
-
-<fd-docs-section-title [id]="'Types'" [componentName]="'select'">
-    Select Border Types
+<fd-docs-section-title [id]="'semantic'" [componentName]="'select'">
+    Select Semantic States
 </fd-docs-section-title>
-<description>An example of the select component with diffrent types of Border type and ability to add icon.
+<description>
+    Use semantic states to visually highlight current state of the Select control.
 </description>
 <component-example>
-	<fd-select-types-example></fd-select-types-example>
+    <fd-select-semantic-state-example></fd-select-semantic-state-example>
 </component-example>
-<code-example [exampleFiles]="selectTypes"></code-example>
+<code-example [exampleFiles]="selectSemantic"></code-example>
 
 <separator></separator>
 
-<fd-docs-section-title [id]="'programmaticControl'" [componentName]="'select'">
-    Programmatic Control
+<fd-docs-section-title [id]="'custom-select-template'" [componentName]="'select'">
+    Custom Control Content
 </fd-docs-section-title>
-
 <description>
-    Controlling the select through programmatic methods is possible by calling the select component instance directly.
-    Note that this can be done through ViewChild as well.
+    The content of the Select control can be customized using a custom template which needs to be referenced in the
+    <code>[controlTemplate]</code> input.<br/>
+    Select provides the following context values to the template:
```

Comment:
Should we move this sentence to line 32 before the break? It looks weird to me.
<img width="820" alt="Screen Shot 2020-04-27 at 9 18 16 AM" src="https://user-images.githubusercontent.com/54723167/80376591-0def8a00-8868-11ea-97b4-d9de3ca58540.png">